dashboard/public/openapi.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.3
 servers:
-  - url: '{schema}://{url}:{port}'
+  - url: "{schema}://{url}:{port}"
```

Comment:
(prettier, not functional changes here)

Reply:
all these changes should be carried out to swaggerhub, no?

Reply:
Yep, once the PR is merged, I'll update the changes in swaggerhub